test/Microsoft.ML.Tests/Transformers/ConvertTests.cs

```diff
@@ -243,7 +243,7 @@ public void TypeConvertKeyBackCompatTest()
         {
             // Model generated using the following command before the change removing Min and Count from KeyType.
             // ML.Transforms.Conversion.ConvertType(new[] { new TypeConvertingEstimator.ColumnInfo("key", "convertedKey",
-            //      DataKind.U8, new KeyCount(4)) }).Fit(dataView);
+            //      ScalarType.Ulong, new KeyCount(4)) }).Fit(dataView);
```

Comment:
`ScalarType` ?

Reply:
Should be `DataKind.UInt64`. Thanks.

---
In reply to: [259522599](https://github.com/dotnet/machinelearning/pull/2661#discussion_r259522599) [](ancestors = 259522599)